src/development/vortex/development/exporter/utils/onnx/graph_ops/base_ops.py

```diff
@@ -0,0 +1,23 @@
+import onnx
+
+class GraphOpsBase:
+    def __init__(self):
+        """Base Class to represents onnx graph ops
+        """
+        pass
+
+    def run(self, model: onnx.ModelProto) -> onnx.ModelProto:
```

Comment:
what do you think about using [`abstractmethod`](https://docs.python.org/3/library/abc.html#abc.abstractmethod)? I just realized we can actually enforce abstract function in python

Reply:
something like this?
```
class Base(metaclass=ABCMeta):
   ...
   @abc.abstractmethod
   def run(...)
```

Reply:
ok please wait

Reply:
yepp